src/Circle.java

```diff
@@ -0,0 +1,10 @@
+public class Circle extends Figures {
+
+    private String FigureName = "Circle";

```

Comment:
All your figures have figureName, you should take it off to the abstract class

Reply:
if you don't want to change FigureName in future you should also use readonly access modifier and initialize this name inside the constructor